test/lib/server/services/qualityControlFlag/QcFlagService.test.js

```diff
@@ -453,15 +453,15 @@ module.exports = () => {
             await assert.rejects(
                 () => qcFlagService.createForSimulationPass(qcFlagCreationParameters, relations),
                 // eslint-disable-next-line max-len
-                new BadParameterError(`Given QC flag period (${new Date('2019-08-08 11:36:40').getTime()}, ${new Date('2019-08-09 05:40:00').getTime()}) is out of run trigger period (${(1565314200 - 45000) * 1000}, ${(1565314200 + 45000) * 1000})`),
+                new BadParameterError(`Given QC flag period (${new Date('2019-08-08 11:36:40').getTime()}, ${new Date('2019-08-09 05:40:00').getTime()}) is out of run (${(1565314200 - 45000) * 1000}, ${(1565314200 + 45000) * 1000}) period`),
```

Comment:
No numbers. I really don't understand how you prefer to add the complexity of run middle point and half duration compared to start/stop...